plugins/vendor/webex/batch/processors/KWebexAPIDropFolderEngine.php

```diff
@@ -34,11 +34,18 @@ class KWebexAPIDropFolderEngine extends KDropFolderFileTransferEngine
 	public function watchFolder(KalturaDropFolder $dropFolder)
 	{
 		$this->initDropFolderEngine($dropFolder);

```

Comment:
i think that we also need to reset the paginationLink on the client class when we init new folder, to make sure there is no case we use a previously set pagainationLink